app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -41,19 +42,36 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               className="w-full flex-grow-0"
               label="Title"
               name="title"
+              placeholder="Enter note title"
             />
             <Textarea
               required
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+              placeholder="Enter note description"
+              rows={2}
+            />
+            <Select
+              required
+              className="w-full flex-grow-0"
+              label="Assigned contact"
+              name="contact"
+              placeholder="Select Role"
+            />
+            <Select
+              required
+              className="w-full flex-grow-0"
+              label="Tags"
+              name="tag"
+              placeholder="Select Tag"
```

Comment:
- Why's that in your business logic you don't allow multiple tags to be selected?
- Also your label and placeholder are having disparity because at one place you are using singular `Tag` and at the other you are using plural `Tags`

Reply:
Pardon me Yedhin. I was oblivious of this. I will rectify this at once.